pyproject.toml

```diff
@@ -50,3 +50,6 @@ copyright-regexp = "Copyright\\s\\d{4}([-,]\\d{4})*\\s+%(author)s"
 ignore_missing_imports = true
 explicit_package_bases = true
 namespace_packages = true
+
+[tool.pylint.SIMILARITIES]
+min-similarity-lines=10
```

Comment:
Why this change? Is it possible to refactor to avoid it?

Reply:
Well it's a false-positive, pylint fails on this because there are several lines that are "found" in two files, the _patched_charm.py and test_charm.py, although in the first file the culprit is on line 61 with `SCRIPT_PATH = __import__("charm").SCRIPT_PATH` that seems to get expanded and pylint thinks it is a duplicate. I'm using this to reduce the noise. If there is another way I'd be happy to try, but this is what I ended up with.

Reply:
The alternative is to disable the rule specifically on the offending line, that way we retain the linting rule on the other parts of the repository

Reply:
There is no offending line, placing the disable on some of the lines quoted in the error does not produce the desired result, even placing the disable on top of the file does not disable it, perhaps due to the import. More on the issue here https://github.com/PyCQA/pylint/issues/214 and also here https://github.com/snapcore/snapcraft/pull/3655